spot-electron/package.json

```diff
@@ -0,0 +1,35 @@
+{
+    "author": "Jitsi",
+    "build": {
+        "appId": "org.jitsi.spot",
+        "mac": {
+            "category": "public.app-category.business",
+            "target": "default"
+        }
+    },
+    "description": "",
+    "devDependencies": {
+        "babel-eslint": "10.0.1",
+        "electron": "5.0.1",
+        "electron-builder": "20.40.2",
+        "eslint": "5.16.0",
+        "eslint-config-jitsi": "github:jitsi/eslint-config-jitsi#1.0.1",
+        "eslint-plugin-flowtype": "3.8.2",
+        "eslint-plugin-import": "2.17.2",
+        "eslint-plugin-jsdoc": "4.8.4"
+    },
+    "license": "ISC",
+    "main": "main.js",
+    "name": "spot-electron",
+    "productName": "Jitsi Spot",
+    "scripts": {
+        "dist": "electron-builder",
+        "lint": "eslint .",
+        "pack": "electron-builder --dir",
+        "start": "electron ."
+    },
+    "version": "1.0.0",
```

Comment:
I wonder if we shouldn't make it v1. I wonder if it makes any difference at all right now.

Reply:
I think this is something yet to sort out. I'm okay with any kind of versioning

Reply:
i just remembered we do date versioning (YYYY.MM.DD) for spot web releases. We should be doing that for mobile as well so probably we'll end up doing that for electron if possible.